gobblin-api/src/main/java/gobblin/configuration/ConfigurationKeys.java

```diff
@@ -744,6 +745,8 @@
    * Hive registration properties
    */
   public static final String HIVE_REGISTRATION_POLICY = "hive.registration.policy";
+  // Set to true if hive registration is done in taskStateTracker on task commit.
+  public static final String PIPELINED_HIVE_REGISTRATION_ENABLED = "hive.registration.pipelined.enabled";
```

Comment:
Remove this option. I think it is clearer if setting TASK_STATE_COLLECTOR_HANDLER_CLASS means a handler is being used. Also, in the future there may be other handler types.